client/app/scripts/liveblog-bloglist/views/main.html

```diff
@@ -64,12 +64,16 @@
                         <dd sd-reldate ng-model="blog._created"></dd>
                         <dt>{{ ::'Last udpated' | translate }}:</dt>
                         <dd><time>{{ ::blog._updated | reldate }}</time></dd>
-                        <dt>{{ ::'Number of Posts' | translate }}:</dt>
+                        <dt>{{ ::'No. of Posts' | translate }}:</dt>
                         <dd>{{ ::blog.posts_count }}</dd>
                         <dl ng-if="blog.last_posted">
                         <dt translate> Last posted</dt>
                         <dd sd-reldate ng-model="blog.last_posted"></dd>
                         </dl>
+                        <dl ng-if="blog.posts_count == 0">
+                        <dt translate> Last posted</dt>
+                        <dd> No posts yet</dd>
```

Comment:
This syntax was "friendly" to me and I didn't want to waste time on finding another one. Is it looking awful?